gitpr/gitpr.go

```diff
@@ -266,7 +266,7 @@ type GitHubRequestError struct {
 }
 
 func PostGitHub(ownerRepo string, request *GitHubRequest, pat string) (response *GitHubResponse, err error) {
-	prSubmitContent, err := json.MarshalIndent(request, "", "")
+	prSubmitContent, _ := json.MarshalIndent(request, "", "")
```

Comment:
Not returning the err is simply a mistake. This is some early Go code before I was in tune with Go sensibilities. 😄 (I also wouldn't use bare `return` here--it's basically always more trouble than it's worth in my experience. No need to change that with this PR, though.)

Reply:
updated PTAL